insta/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Comments = props =>{
+    return(
+        <div>
```

Comment:
You could also try using fragments `<>` instead of a div to wrap the child components